charts/linkerd2/values.yaml

```diff
@@ -125,3 +125,9 @@ ProxyInjectDisabled: disabled
 ControllerComponentLabel: linkerd.io/control-plane-component
 ControllerNamespaceLabel: linkerd.io/control-plane-ns
 LinkerdNamespaceLabel: linkerd.io/is-control-plane
+
+# If the namespace is controlled by an external tool
+# or can't be installed with Helm,
+# disable its installation
+# https://github.com/linkerd/linkerd2/issues/3412
+InstallNamespace: true
```

Comment:
Could you add these two notes in the comments:
- The namespace created by the external tool must match the `Namespace` value above
- The external tool needs to create the namespace with the label `linkerd.io/is-control-plane: "true"`